### DISCUSSION TITLE
What is the learning rate?

### DISCUSSION THREAD

**Discussion by BUETianBro** [BUETianBro/Python-AI-QnA#29] [Q&A] [answered]
What is the learning rate?

**Comment by almuzahidseyam** [comment, marked as answer]
A hyperparameter that controls how much to change the model weights in response to the estimated error.
